src/components/Carousel/Carousel.tsx

```diff
@@ -145,10 +147,10 @@ export const Carousel: FC<CarouselProps> = ({
     >
       <ScrollContainer
         className={twMerge(theme.scrollContainer.base, (isDeviceMobile || !isDragging) && theme.scrollContainer.snap)}
-        draggingClassName="cursor-grab"
+        draggingClassName={draggable ? 'cursor-grab' : ''}
```

Comment:
I think that you can use `null` instead of `''`

Reply:
Actually, I think that you should not change the `draggingClassName` at all, but disable the `vertical` property from the `ScrollContainer`:

https://www.npmjs.com/package/react-indiana-drag-scroll#component-properties